### ISSUE TITLE
TF 2.0: Move TensorFlow Lite out of tf.contrib.

### ISSUE THREAD

**Issue by dynamicwebpaige** [tensorflow/tensorflow#25373]
**[TensorFlow Lite](https://www.tensorflow.org/lite/)** is the official solution for running machine learning models on mobile and embedded devices. It enables on‑device machine learning inference with low latency and a small binary size on Android, iOS, and other operating systems.

In TensorFlow 1.x, TF Lite was housed in `tf.contrib`, a somewhat volatile location. With TF 2.0, we intend to migrate TF Lite to the main API, under [`tf.lite`](https://www.tensorflow.org/versions/r2.0/api_docs/python/tf/lite).

**Comment by angersson** [comment]
This was completed at the end of October. Here's a copy of the relevant email on [tflite@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!searchin/tflite/moving%7Csort:date/tflite/iIIXOTOFvwQ/EwfiRFtvAQAJ):
* * * 
TL;DR TensorFlow Lite will move from tensorflow/contrib/lite to tensorflow/lite on Wednesday, Oct. 31. This should be a no-op for Python package users (but please update your usage) and will break Bazel users.

Hello everyone,

TensorFlow Lite is moving from //tensorflow/contrib/lite to //tensorflow/lite. The move is scheduled for this Wednesday, October 31, and will take effect on the master TensorFlow branch, in upcoming nightly packages, and in TensorFlow 1.13+. This email describes what you can expect from the move.

We previously announced that the migration would happen on Wednesday Oct. 10, but we discovered blockers that needed a significant change in direction. Sorry about the confusion.

For anyone who depends on tf.contrib.lite in TensorFlow's Pip package, we’ve added import helpers so that tf.contrib.lite points to tf.lite. This will affect upcoming nightly packages and TF 1.13+. tf.contrib will not be present in TF 2.0. Please update your imports as soon as you can.

For anyone who depends on TF Lite’s OSS Bazel BUILD structure or Makefiles, you will need to update packages that reference tensorflow/contrib/lite so that they point to tensorflow/lite instead.
